subprojects/junit5/junit5.gradle

```diff
@@ -0,0 +1,14 @@
+description = "Mockito preconfigured inline mock maker (intermediate and to be superseeded by automatic usage in a future version)"
+
+apply from: "$rootDir/gradle/java-library.gradle"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
Needs to be 1.6

Reply:
I am afraid that don't work cause JUnit5 ExtensionContext.getTestInstance() returns Optional that was introduced in Java 8. IntelliJ refuses shows an error in this case.